.gitignore

```diff
@@ -6,5 +6,9 @@
 .gitignore
 .prettierrc
 
+docker/cypress/downloads
+docker/cypress/videos
+docker/cypress/screenshots
```

Comment:
can we add the `cypress.env.json` file as well if we need configured env vairables